### PR TITLE
DC.5 Allocation of Grimes CSR strategy for FT's

### PR DESCRIPTION
**Pull request by TanyaS08**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -16,11 +16,6 @@
 .DS_Store
 *.DS_Store
 
-# data files
-.xls
```
Comment (TanyaS08) [inline .gitignore:20]: Why did you remove these???

File: scripts/DC5_CSR_export_for_excel_StrateFy_classification.R [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+traits_wide <-
+  traits %>% 
+  pivot_wider(id_cols = -c(trait,value),
+              names_from = trait,
+              values_from = value,
+              values_fill = NA)
+
+# Create average trait LDMC, Leaf area and SLA for each species
+# Not taking into account burn history, or elevational variation
+summary_trait_species <-
```
Comment (TanyaS08) [inline scripts/DC5_CSR_export_for_excel_StrateFy_classification.R:45]: Oh this is great - love it!

**Review by TanyaS08 — COMMENTED** [review]
Stellar work

File: .gitignore (hunk 1 above)
Comment (srg101) [reply .gitignore:20]: I listen to the boss :P

**Comment by TanyaS08** [comment]
I've changed the folder structure so now all data files needed for the CSR work is in data/CSR - amended traits (i.e. final output) are still in data/processed though. data/raw is now back on the .gitignore

**Review by srg101 — APPROVED** [review]
Seems ok

File: scripts/DC5_CSR_trait_merge.R [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+# Merge back with original traits_wide dataset, based on taxon ID.
+traits_csr <- left_join(traits_wide, csr_taxon_subset, by="taxon") %>% 
+  pivot_longer(cols = c(dry_mass_g, ldmc, leaf_area_cm2, leaf_thickness_mm,
+                        plant_height_cm,
+                        sla_cm2_g, wet_mass_g),
+               names_to = 'trait',
+               values_to = 'value')
+
+dir.create("data/processed")
+write.csv(traits_csr, file = "data/processed/traits_csr_wide.csv")
```
Comment (TanyaS08) [inline scripts/DC5_CSR_trait_merge.R:43]: Just checking here we are calling the output 'wide' but in l. 36 you pivot the data into long format. maybe we should change the output name? I feel like going with `traits_csr.csv` should be good enough?

**Review by TanyaS08 — COMMENTED** [review]
Can we maybe add a short section to the README as to how this code 'works' _i.e_ how one should go about running/implementing it?
